### PR TITLE
Master mail finish tde

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by dreispt** [comment]
@tde-banana-odoo Any changes here to improve the low performance of the record rules limiting access to documents where the User is a Follower? 
Is `auto_join` an option?

File: addons/link_tracker/controller/main.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 import werkzeug
 
-from openerp import http
-from openerp.http import request
+from odoo import http
+from odoo.http import request
 
 
-class link_tracker(http.Controller):
+class LintTracker(http.Controller):
```
Comment (dreispt) [inline addons/link_tracker/controller/main.py]: s/Lint/Link

Comment (tde-banana-odoo) [inline addons/link_tracker/controller/main.py]: :)

**Comment by tde-banana-odoo** [comment]
Well this is not the purpose of this branch ... Did you already perform some tests to see if it could be useful ?

**Comment by dreispt** [comment]
I just noticed that the data model changed quite a bit.
AFAICS now the record rules use `channel_partner_ids` instead of `message_partner_ids`, and there are some extra indices.
To test I would need to load a lot of data, so nothing that I can do easily.
In due moment I'll try that. 😸

**Comment by dreispt** [comment]
PRs on master don't show a build status?

**Pull request merged**